### PR TITLE
Document generating ctags tests using the pretty-printing script

### PR DESCRIPTION
**Pull request by techee**

### CODE REVIEW

File: HACKING [hunk 1]
```diff
@@ -699,9 +699,13 @@ It can be either long or short, depending on what it tests.
...
-should be in the format generated by ``geany -g``. This file contains
-the tag information expected to be generated from the corresponding
-source file.
+should be in the format generated by::
+
+    $ geany -g tmp.ext.tags tests/ctags/mytest.ext
+    $ cat tmp.ext.tags | scripts/print-tags.py > tests/ctags/mytest.ext.tags
+
+This file contains the tag information expected to be generated from
+the corresponding source file together with its human-redable form.
```
Comment (elextr) [inline HACKING:708]: "readable" not "redable" otherwise fine

Comment (techee) [reply HACKING]: Yeah, thanks, fixed.

**Comment by techee** [comment]
Will give this PR one week in case there are some comments before merging it.

File: HACKING [hunk 2]
```diff
@@ -699,9 +699,13 @@ It can be either long or short, depending on what it tests.
 
 The expected output should be in the file ``tests/ctags/mytest.ext.tags``
 (which is the same name as the source, but with ``.tags`` appended), and
-should be in the format generated by ``geany -g``. This file contains
-the tag information expected to be generated from the corresponding
-source file.
+should be in the format generated by::
+
+    $ geany -g tmp.ext.tags tests/ctags/mytest.ext
+    $ cat tmp.ext.tags | scripts/print-tags.py > tests/ctags/mytest.ext.tags
```
Comment (b4n) [inline HACKING:705]: nitpick: could use `scripts/print-tags.py < tmp.ext.tags > tests/ctags/mytest.ext.tags` and drop the :cat:

**Review by b4n — APPROVED** [review]
LGTM otherwise

File: HACKING (hunk 2 above)
Comment (techee) [reply HACKING]: Right. Despite the fact that 🐈s are great, this one can be dropped. I'll merge the 🐱less version.

**Pull request merged**